spec/jobs/integration_spec.rb

```diff
@@ -8,6 +8,8 @@
   let(:hash) do
     { druid: druid, version: version, endpoint: endpoint, checksum: checksum }
   end
+  let(:object) { instance_double(Aws::S3::Object, exists?: false, put: true) }
```

Comment:
another opportunity to use `s3_object` instead of `object`

Reply:
in the context of these tests, i think this is fine